File: BackEnd/backend/src/main/java/com/fitapp/backend/application/dto/exercise/ExerciseRequest.java
package com.fitapp.backend.application.dto.exercise;

import jakarta.validation.constraints.NotNull;
import lombok.Data;

@Data
public class ExerciseRequest {
    @NotNull
    private Long exerciseId;
    @NotNull
    private Integer sets;
    private Integer targetReps;
    private Double targetWeight;
    private Integer restIntervalSeconds;
}

File: FrontEnd/app/src/main/java/com/fitapp/appfit/response/exercise/ExerciseRequest.kt
package com.fitapp.appfit.response.exercise

data class ExerciseRequest(
    val exerciseId: Long,
    val sets: Int,
    val targetReps: String,
    val targetWeight: Double?,
    val restIntervalSeconds: Int?,
    val position: Int,
    val notes: String?
)